app/jobs/sync_channel_promo_registrations_stats_job.rb

```diff
@@ -0,0 +1,10 @@
+# Fetches and saves the referral stats for channel owned codes
+class SyncChannelPromoRegistrationsStatsJob < ApplicationJob
+  include PromosHelper
+  queue_as :scheduler
+
+  def perform(promo_id: active_promo_id)
+    promo_registrations = PromoRegistration.channel
```

Comment:
Did you want to lookup `active_promo_id`?

Reply:
Hm, the `active_promo_id` was there to be able to distinguish between different promotions.  We originally we're going to run multiple referral promos, but in practice we're only running one long one, so the `active_promo_id` should be the same for all codes.

I think @evq may be using `active_promo_id` to distinguish between unattached and channel owned referral codes, but I don't think that matters here.

Reply:
I did not and it is removed now, thanks.